app/src/main/AndroidManifest.xml

```diff
@@ -90,14 +100,6 @@
             <meta-data
                 android:name="android.support.PARENT_ACTIVITY"
                 android:value="com.intervaltimer.google.activities.MainActivity" />
-            <intent-filter android:autoVerify="true">
```

Comment:
Please squash your commits before opening the pull request if some commits modify previous commits